requirements.txt

```diff
@@ -55,7 +55,7 @@ beautifulsoup4
 lxml >=4.8.0,<6
 
 # XPath 2.0-3.1 support - 4.2.0 broke something?
-elementpath==4.1.5
+elementpath==4.4.0
```

Comment:
Is time to upgrade?

Reply:
> Is time to upgrade?

Sure, if the tests pass it's OK

Reply:
this change was required to fix this PR?

Reply:
Since the newer PR(https://github.com/dgtlmoon/changedetection.io/pull/2351) uses fragment=True option, >=4.1.5 won't work. and 4.2.0 has another problem. So minimum is 4.2.1